apps/web/src/server/views/applications/create/check-your-answers.njk

```diff
@@ -0,0 +1,203 @@
+{% extends "app/layouts/app.layout.njk" %}
```

Comment:
no reason for this to appear 'green', however I did make some changes:
- renamed from _check-your-answers to check-your-answers
- extending from app.layout rather than from application-create-case (which was deleted)